shared/contracts/BaseTemplate.sol

```diff
@@ -235,8 +238,7 @@ contract BaseTemplate is APMNamehash, IsContract {
 
     function _createTokenManagerPermissions(ACL _acl, TokenManager _tokenManager, address _grantee, address _manager) internal {
         _acl.createPermission(_grantee, _tokenManager, _tokenManager.MINT_ROLE(), _manager);
-        _acl.createPermission(_grantee, _tokenManager, _tokenManager.ASSIGN_ROLE(), _manager);
-        _acl.createPermission(_grantee, _tokenManager, _tokenManager.REVOKE_VESTINGS_ROLE(), _manager);
```

Comment:
On what criteria are we assigning and not assigning some of the roles here? The roles related to vesting will be particularly useful when integrating the Presale and/or the Fundraising apps.

Reply:
This would be the default helper, I basically assumed the most reusable stuff based on the templates we have now